OpenRA.Mods.RA/Widgets/Logic/LobbyUtils.cs

```diff
@@ -350,8 +351,7 @@ public static void SetupKickSpectatorsWidget(Widget parent, OrderManager orderMa
 				orderManager.IssueOrders(
 					orderManager.LobbyInfo.Clients.Where(
 						c => c.IsObserver && !c.IsAdmin).Select(
-							client => Order.Command(String.Format("kick {0} {1}", client.Index, client.Name
-							))).ToArray());
+							client => Order.Command(string.Format("kick {0} {1}", client.Index, client.Name))).ToArray());
```

Comment:
It would be even better if this was changed to `"kick {0} {1}".F(client.Index, client.Name)` instead.